index.php

```diff
@@ -2,6 +2,39 @@
 $is_auth = rand(0, 1);
 
 $user_name = 'Алексей'; // укажите здесь ваше имя
+
+$articles = [
+    ['title' => 'Цитата',
+     'type' => 'post-quote',
+     'content' => 'Мы в жизни любим только раз, а после ищем лишь похожих',
+     'user_name' => 'Лариса',
+     'avatar' => 'userpic-larisa-small.jpg'
```

Comment:
В последнем элементе массива желательно тоже добавлять запятую
```suggestion
     'avatar' => 'userpic-larisa-small.jpg',
```
это не создаст еще один пустой элемент, зато позволит проще потом работать с git при работе над одним файлом нескольких разработчиков